modules/View.js

```diff
@@ -56,6 +56,72 @@
       if (!options.noActivate) {
         this.activate();
       }
+      // this.updateDelegateEvents();
+    },
+
+
+    /**
+    * called by updateDelegateEvents to wrap callback functions of event calls
+    * with event triggers
+    * @param method {function} callback method of event
+    * @param eventName {String} description of event type 
+    * @return modified mehtod with before/after event triggers
+    * @method trackEvents
+    */
+    trackEvents : function(method, eventName){
+      var self = this;
+      var methodCopy = method;
+      method = _.bind(function(){
+        var uuid = (new Date()).getTime().toString(16)+Math.floor(1E7*Math.random()).toString(16);
+        this.trigger('clickEventTiming', {
+          uuid: uuid,
+          type: 'clickEvent',
```

Comment:
the type here is incorrect. DOM event callbacks can be on any trigger: 'click', 'hover', 'mouseup'. They're all DOM events though